src/Views/batchprocessor.html

```diff
@@ -12,6 +12,11 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
 <!-- Start modal body-->
 <div class="modal-body">
 
+    <!-- Warning Message -->
+    <div id="warningMessageContent" ng-if="!vm.showWarning">
+        <div style="color: red;" ng-bind-html="vm.convertUnsafe(vm.warningMessage)"></div>

```

Comment:
Thought on using the wim-warning class?

```suggestion
        <div class="wim-warning" ng-bind-html="vm.convertUnsafe(vm.warningMessage)"></div>
```